### PR TITLE
Dodnz/announce

### PR DESCRIPTION
**Pull request by DefSol**
Added announcement dates for DoD Auckland - updated logo and banner images

### CODE REVIEW

File: content/events/2017-auckland/index.md [hunk 1]
```diff
@@ -5,92 +5,101 @@ aliases = ["/events/2017-auckland/welcome"]
...
-Check out last year's conference in Wellington
-</p>
+   <p>We are excited to bring DevOpsDays to Auckland this year that will bring contributors from dev, ops, infoSec, QA management, leadership and a range of cultures - to share, learn, laugh and be encourage by the many great things happening in New Zealand technology sector.</p>
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/JuctqYgeyiI" frameborder="0" allowfullscreen></iframe>
-
-<div class = "row">
-  <div class = "col-md-2">
-    <strong>Dates</strong>
+  <p>We are in  bigger facility and are expecting 300 people to take part in the event. Our CFP will be opening within the week, with early bird tickets to go on sale in the next week as well.
```
Comment (bridgetkromhout) [inline content/events/2017-auckland/index.md]: a bigger facility

**Comment by bridgetkromhout** [comment]
Hi, @DefSol - I noticed that you're pointing to a different email on your contact page. We require it to point to organisers-auckland-2017@devopsdays.org - please correct it. Thanks.

**Comment by bridgetkromhout** [comment]
Overall, looks great! We do need to point to the @devopsdays.org email (so that the core team can stay in the loop). When that's corrected, this can be merged!

**Comment by DefSol** [comment]
Hey @bridgetkromhout my bad - have corrected the address using the english spelling, I'm sure @MikeRosTX setup an alias for the US spelling as well. Also word smithed the last paragraph on the index content page a little...

**Comment by bridgetkromhout** [comment]
Great! And yes, there is an alias so either spelling will work.

**Pull request merged**